src/popup/components/ConsensusLabel.vue

```diff
@@ -16,6 +16,7 @@ export default defineComponent({
     confirmationsRequired: { type: Number, required: true },
     defaultConfirmedBy: { type: Number, default: 0 },
     hasPendingTransaction: Boolean,
+    hasDot: Boolean,
```

Comment:
This looks rather funny :) I suggest to align with Tsvetan if this component should end with dot or not.